test/Sentry.Maui.Tests/SentryMauiScreenshotTests.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.Extensions.Options;
+using Sentry.Internal.Http;
+using Sentry.Maui.Internal;
+
+namespace Sentry.Maui.Tests;
+
+public class SentryMauiScreenshotTests
+{
+    private class Fixture
+    {
+        public MauiAppBuilder Builder { get; }
+        public FakeTransport Transport { get; private set; } = new FakeTransport();
+        public InMemoryDiagnosticLogger Logger { get; private set; } = new InMemoryDiagnosticLogger();
+
+        public Fixture()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder.Services.AddSingleton(Substitute.For<IApplication>());
+
+            builder.Services.Configure<SentryMauiOptions>(options =>
+            {
+                options.Transport = Transport;
+                options.Dsn = ValidDsn;
+                options.AttachScreenshot = true;
+                options.Debug = true;
+                options.DiagnosticLogger = Logger;
+                options.AutoSessionTracking = false; //Get rid of session envelope for easier Assert
+                options.CacheDirectoryPath = null;   //Do not wrap our FakeTransport with a caching transport
+            });
+
+            Builder = builder;
+        }
+    }
+
+    private readonly Fixture _fixture = new();
+
+    [Fact]
+    public async Task CaptureException_WhenAttachScreenshots_ContainsScreenshotAttachmentAsync()
+    {
+        // Arrange
+        var builder = _fixture.Builder.UseSentry();
+
+        // Act
+        using var app = builder.Build();
+        var client = app.Services.GetRequiredService<ISentryClient>();
+        var sentryId = client.CaptureException(new Exception());
+        await client.FlushAsync();
+
+        var options = app.Services.GetRequiredService<IOptions<SentryMauiOptions>>().Value;
+        
+        var envelope = _fixture.Transport.GetSentEnvelopes().FirstOrDefault(e => e.TryGetEventId() == sentryId);
+        envelope.Should().NotBeNull();
+
+        var envelopeItem = envelope!.Items.FirstOrDefault(item => item.TryGetType() == "attachment");
+
+        // Assert
+#if __MOBILE__
+        if (_fixture.Logger.Entries.Any(entry => entry.Level == SentryLevel.Error && entry.Exception is NullReferenceException))
```

Comment:
This is ignoring that flaky issue on Android, right? so please be clearn:

```suggestion
        #if ANDROID
        // On Android this can fail due to XYZ, see issue https://github.com/maui/etc
        if (_fixture.Logger.Entries.Any(entry => entry.Level == SentryLevel.Error && entry.Exception is NullReferenceException))
```